src/Orc.Controls/Orc.Controls.Shared/Controls/DateRangePicker/Views/DateRangePickerControl.xaml

```diff
@@ -0,0 +1,55 @@
+﻿<catel:UserControl x:Class="Orc.Controls.DateRangePickerControl"
```

Comment:
I want to start getting rid of the Control postfixes in Orc.Controls. Let's start with this control, I just want to write this in xaml:

```
<orc:DateRangePicker StartDate="{Binding }" EndDate="{Binding }" Span="{Binding }" />
```